Userland/Applications/PixelPaint/Tools/MoveTool.h

```diff
@@ -21,12 +21,14 @@ class MoveTool final : public Tool {
     virtual void on_mousemove(Layer*, MouseEvent&) override;
     virtual void on_mouseup(Layer*, MouseEvent&) override;
     virtual void on_keydown(GUI::KeyEvent&) override;
-    virtual Variant<Gfx::StandardCursor, NonnullRefPtr<Gfx::Bitmap>> cursor() override { return Gfx::StandardCursor::Move; }
+    virtual Variant<Gfx::StandardCursor, NonnullRefPtr<Gfx::Bitmap>> cursor() override;
 
 private:
     RefPtr<Layer> m_layer_being_moved;
     Gfx::IntPoint m_event_origin;
     Gfx::IntPoint m_layer_origin;
+    bool m_scaling = false;
+    bool m_is_in_lower_right_corner = false;
```

Comment:
I prefer the name `m_mouse_in_resize_corner` or something along these lines, to make clear _what_ is in _which_ corner and for _what function_.